runtime/assembly/__tests__/mock-builder.ts

```diff
@@ -56,8 +56,25 @@ export namespace MockBuilder {
     /**
      * Returns SCALE encoded Inherent Mock and Instance of that Mock
      */
-     export function getInherentMock(): MockResult<Inherent> {
-         return new MockResult(MockHelper._getEmptyInherentInstance(), MockConstants.DEFAULT_INHERENT);
+     export function getInherentMock(): MockResult<InherentData> {
+        const DEFAULT_INHERENT: u8[] = [
```

Comment:
Actually, I've added a new file called `mock-constants.ts`. In that file, all of the byte array mocks are placed. I though that it would be better that way, because we have every byte mock in one file and the code here for getting the mocks is cleaner.
Could you place this array there as-well?